### PR TITLE
Port Nushell activation scripts to 0.67

### PR DESCRIPTION
**Pull request by kubouch**
### Thanks for contributing, make sure you address all the checklists (for details on how see [development documentation](https://virtualenv.pypa.io/en/latest/development.html#development))!

The Nushell release 0.67 comes with a new variable naming convention which breaks the (de)activation scripts.

- [ ] ran the linter to address style issues (`tox -e fix_lint`)
- [x] wrote descriptive pull request text
- [ ] ensured there are test(s) validating the fix
- [x] added news fragment in `docs/changelog` folder
- [ ] updated/extended the documentation

### CODE REVIEW

**Comment by gaborbernat** [comment]
Will this still work on older versions?

**Comment by jntrnr** [comment]
> Will this still work on older versions?

Yes, it should

**Comment by kubouch** [comment]
> Will this still work on older versions?

Yes, it should work with any 0.6x Nushell version.

**Comment by gaborbernat** [comment]
Can you add a changelog entry please?

File: docs/changelog.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Features - 20.16.0
 - Drop support for running under Python 2 (still can generate Python 2 environments) - by :user:`gaborbernat`. (`#2382 <https://github.com/pypa/virtualenv/issues/2382>`_)
 - Upgrade embedded pip to ``22.2`` from ``22.1.2`` and  setuptools to ``63.2.0`` from ``62.6.0`` -
   by :user:`gaborbernat`. (`#2383 <https://github.com/pypa/virtualenv/issues/2383>`_)
-
+- Update Nushell activation scripts to version 0.66 - by :user:`kubouch`.
```
Comment (gaborbernat) [inline docs/changelog.rst:16]: Not here... https://virtualenv.pypa.io/en/latest/development.html#news-entries

**Review by gaborbernat — CHANGES_REQUESTED** [review]

File: docs/changelog.rst (hunk 1 above)
Comment (kubouch) [reply docs/changelog.rst]: I hope I did it correctly this time.

File: docs/changelog/2386.misc.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Update Nushell activation scripts to version 0.66 - by :user:`kubouch`.
```
Comment (gaborbernat) [inline docs/changelog/2386.misc.rst:1]: This is a feature, not misc 👍 Supporting a new version of a shell is a feature.

**Review by gaborbernat — APPROVED** [review]

**Comment by kubouch** [comment]
Small change: The Nushell version this PR is addressing is 0.67, not 0.66. Either way, it shouldn't break anything, just ensures the activation scripts will be working in the future.

**Pull request merged**